array.go

```diff
@@ -20,6 +20,7 @@ var typeSQLScanner = reflect.TypeOf((*sql.Scanner)(nil)).Elem()
 //
 // For example:
 //  db.Query(`SELECT * FROM t WHERE id = ANY($1)`, pq.Array([]int{235, 401}))
+//  db.Query(`SELECT * FROM t WHERE id = ANY($1)`, []int{235, 401}) // go1.9+
```

Comment:
The tests are all `[]int64`. Does this example work? If there is a limitation related to `driver.Valie`, it should probably be documented.